src/coll/options.rs

```diff
@@ -268,12 +268,16 @@ pub struct DeleteOptions {
     /// The write concern for the operation.
     #[builder(default)]
     pub write_concern: Option<WriteConcern>,
+
+    /// The index to use for the operation.
```

Comment:
This and all the other hints should be documented as only being supported in MongoDB 4.4+.